lib/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -409,6 +403,21 @@ public JsonObject asJSON() {
 	 * @param options
 	 */
 	public TokenDetails authorize(TokenParams params, AuthOptions options) throws AblyException {
+		return authorize(params, options, true);
+	}
+
+	/**
+	 * Alias of authorize() (0.9 RSA10l)
+	 */
+	public TokenDetails authorise(TokenParams params, AuthOptions options) throws AblyException {
+		Log.w(TAG, "authorise() alias will be removed in 1.0");
+		return authorize(params, options);
+	}
+
+	/**
+	 * Internal authorize() where the third param chooses whether to force reauthorization.
+	 */
+	public TokenDetails authorize(TokenParams params, AuthOptions options, boolean force) throws AblyException {
```

Comment:
Hmm, maybe. It does need to be public as it is called from Http, but a different name might be good.